tests/model/test_model.py

```diff
@@ -2033,10 +2034,10 @@ def test_toggle_stream_pinned_status(
         model.client.update_subscription_settings.assert_called_once_with(request)
 
     @pytest.mark.parametrize(
-        "narrow, event, called",
+        "narrow, event, is_animation_in_progress, is_recipient_actually_typing",
```

Comment:
While you're at it, let's use cases (`from pytest import param as case`), so that it's evident that which test case pairs with which test id.

Reply:
:+1: